lib/array_intersection.rb

```diff
@@ -1,4 +1,19 @@
 def intersection(list1, list2)
-  raise NotImplementedError, "Intersection not implemented"
+  hash = {}
```

Comment:
I realize I didn't model this well in class, but naming this variable better would be good.